src/Elewant/AppBundle/Resources/views/Default/about.html.twig

```diff
@@ -14,30 +14,34 @@
                     <i class="fa fa-circle fa-stack-2x text-primary"></i>
                     <i class="fa fa-user fa-stack-1x fa-inverse"></i>
                 </span>
-                <h4 class="service-heading">1. Start a Herd</h4>
-                <p class="text-muted">If you own ElePHPants and want to show the world, create your herd and set up which ones you have. It's-a-simple!</p>
+                <h4 class="service-heading">{{ 'site.about.step-1-heading'|trans }}</h4>
+                <p class="text-muted">{{ 'site.about.step-1-subtext'|trans }}</p>
             </div>
             <div class="col-md-4">
                 <span class="fa-stack fa-4x">
                     <i class="fa fa-circle fa-stack-2x text-primary"></i>
                     <i class="fa fa-users fa-stack-1x fa-inverse"></i>
                 </span>
-                <h4 class="service-heading">2. Offer and Request ElePHPants</h4>
-                <p class="text-muted">Coming soon: After displaying your magnificent collection, you select specimens that you wish to trade and specimens that you would like to add to your herd.</p>
+                <h4 class="service-heading">{{ 'site.about.step-2-heading'|trans }}</h4>
+                <p class="text-muted">{{ 'site.about.step-2-subtext'|trans }}</p>
             </div>
             <div class="col-md-4">
                 <span class="fa-stack fa-4x">
                     <i class="fa fa-circle fa-stack-2x text-primary"></i>
                     <i class="fa fa-handshake-o fa-stack-1x fa-inverse"></i>
                 </span>
-                <h4 class="service-heading">3. Trade!</h4>
-                <p class="text-muted">Coming soon: We will attempt to show you where the closest trade options are, and make it easy for you to get in touch. We hope to capture the history of ElePHPant travels through time...</p>
+                <h4 class="service-heading">{{ 'site.about.step-3-heading'|trans }}</h4>
+                <p class="text-muted">{{ 'site.about.step-3-subtext'|trans }}</p>
             </div>
         </div>
 
         <div class="row text-center">
             <div class="col-lg-12 text-center">
-                <p class="text-muted small">* Read about the <a href="{{ path('history') }}">history of this project</a></p>
+                <p class="text-muted small">
+                    {% trans with {'%link%': '<a href="' ~ path('history') ~ '">' ~ 'site.about.history-link-text'|trans ~ '</a>'} %}
```

Comment:
I see you're using `{% trans with { ... } %}` here, instead of `{{ '...'|trans({ ... }) }}`. I'm curious to why 🙂 Personal style? Some convention?

Reply:
The trans filter escapes by default, the trans tag doesn't. So I didn't fancy piping the translation into `raw` in this case.

It seemed slightly cleaner to me this way, in this particular case. But I switch between the two styles depending on . I don't want to lock done a "one true way to do translations" for the entire project...

Reply:
> The trans filter escapes by default, the trans tag doesn't.

Ah! I didn't know that. 👍 

And agreed, I wasn't out to "lock things down" in this case, I was simply curious 🙂